jwql/instrument_monitors/common_monitors/dark_monitor.py

```diff
@@ -0,0 +1,955 @@
+        #logging.info('Amplifier boundaries: {}'.format(amp_bounds))
+
+        # Calculate mean and stdev values, and fit a Gaussian to the
+        # histogram of the pixels in each amp
+        (amp_mean, amp_stdev, gauss_param, gauss_chisquared, double_gauss_params, double_gauss_chisquared,
+            histogram, bins) = self.stats_by_amp(slope_image, amp_bounds)
+
+        # Construct new entry for dark database table
+        for key in amp_mean.keys():
+            dark_db_entry = {'aperture': self.aperture, 'amplifier': key, 'mean': amp_mean[key],
+                             'stdev': amp_stdev[key],
+                             'source_files': file_list,
+                             'gauss_amplitude': list(gauss_param[key][0]),
+                             'gauss_peak': list(gauss_param[key][1]),
+                             'gauss_width': list(gauss_param[key][2]),
+                             'gauss_chisq': gauss_chisquared[key],
+                             'double_gauss_amplitude1': double_gauss_params[key][0],
+                             'double_gauss_peak1': double_gauss_params[key][1],
+                             'double_gauss_width1': double_gauss_params[key][2],
+                             'double_gauss_amplitude2': double_gauss_params[key][3],
+                             'double_gauss_peak2': double_gauss_params[key][4],
+                             'double_gauss_width2': double_gauss_params[key][5],
+                             'double_gauss_chisq': double_gauss_chisquared[key],
+                             'mean_dark_image_file': mean_slope_file,
+                             'hist_dark_values': bins,
+                             'hist_amplitudes': histogram,
+                             'entry_date': datetime.datetime.now()
+                             }
+            self.stats_table.__table__.insert().execute(dark_db_entry)
+
+    def save_mean_slope_image(self, slope_img, stdev_img, files):
+        """Save the mean slope image and associated stdev image to a file
+
+        Parameters
+        ----------
+        slope_img : numpy.ndarray
+            2D array containing the mean slope image
+
+        stdev_img : numpy.ndarray
+            2D array containing the stdev image associated with the mean
+            slope image.
+
+        files : list
+            List of input files used to construct the mean slope image
+
+        Returns
+        -------
+        output_filename : str
+            Name of fits file to save mean and stdev images within
+        """
+        output_filename = '{}_{}_{}_to_{}_mean_slope_image.fits'.format(self.instrument.lower(),
+                                                                        self.aperture.lower(),
+                                                                        self.query_start, self.query_end)
+        mean_slope_dir = os.path.join(get_config()['outputs'], 'monitor_darks/mean_slope_images/')
```

Comment:
@bhilbert4 It would be slightly safer/more consistent to use:

```python
mean_slope_dir = os.path.join(get_config()['ouputs'], 'monitor_darks', 'mean_slope_images') 
```

Reply:
Updated